disnake/app_commands.py

```diff
@@ -161,7 +161,7 @@ def localize(self, store: LocalizationProtocol) -> None:
 class Option:
     """Represents a slash command option.
 
-    Parameters
+    Attributes
```

Comment:
the parameter and attribute types don't all match in this class:
- `name` and `description` are `str`s, and the `versionchanged` no longer applies
- `choices` gets mapped to a `List[OptionChoice]`

At some point we may want to look into documenting *both* `Parameters` and `Attributes` here, but for now only listing attributes is fine.

Reply:
Mh i think it should be ok to document them both now, otherwise it could be worse later...

Reply:
Hm, fair enough -- it's somewhat rare to document both `Parameters` and `Attributes` sections in the lib, but definitely not the first time (`Activity` types do it as well). Having both would make sense here, sure